<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>761</width>
      <height>511</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Ajouter client</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>60</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>CIN</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>100</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>NOM</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>140</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>PRENOM</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>180</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>NUM TELEPHONE</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>220</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>EMAIL</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="NOM">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>100</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="PRENOM">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>140</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="NUM_TELEPHONE">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>180</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="EMAIL">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>220</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>270</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>270</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="CIN">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>60</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Afficher client</string>
     </attribute>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>661</width>
        <height>241</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>330</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Trier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>330</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Remise</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>330</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Historique</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Chatbox">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>330</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Chatbox</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Supprimer client</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>240</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>140</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>CIN</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="sup">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>140</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>sécurité</string>
     </attribute>
     <widget class="QPushButton" name="optionspushButton_8">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>40</y>
        <width>651</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Options</string>
      </property>
     </widget>
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>80</y>
        <width>651</width>
        <height>381</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>649</width>
         <height>379</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="review">
     <attribute name="title">
      <string>Arduino</string>
     </attribute>
     <widget class="QLineEdit" name="review_arduino">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>120</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="ok">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>220</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>review</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>120</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>review</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>816</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
